eslzArm/eslzArm.json

```diff
@@ -2737,6 +2761,60 @@
                 }
             }
         },
+        {
+            // Assigning decommissioned initiative to decommissioned management group if condition is true
+            "condition": "[equals(parameters('enableDecommissioned'), 'Yes')]",
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2020-10-01",
+            "name": "[variables('deploymentNames').decommissionLzPolicyDeploymentName]",
+            "scope": "[variables('scopes').decommissionedManagementGroup]",
+            "location": "[deployment().location]",
+            "dependsOn": [
+                "policyCompletion"
+            ],
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "contentVersion": "1.0.0.0",
+                    "uri": "[variables('deploymentUris').decommissionLzPolicyAssignment]"
+                },
+                "parameters": {
+                    "topLevelManagementGroupPrefix": {
+                        "value": "[parameters('enterpriseScaleCompanyPrefix')]"
+                    },
+                    "enforcementMode": {
+                        "value": "Default"
+                    }
+                }
+            }
+        },
+        {
+            // Assigning sandbox initiative to sandbox management group if condition is true
+            "condition": "[equals(parameters('enableSandbox'), 'Yes')]",
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2020-10-01",
+            "name": "[variables('deploymentNames').sandboxLzPolicyDeploymentName]",
+            "scope": "[variables('scopes').sandboxManagementGroup]",
+            "location": "[deployment().location]",
+            "dependsOn": [
+                "policyCompletion"
+            ],
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "contentVersion": "1.0.0.0",
+                    "uri": "[variables('deploymentUris').sandboxLzPolicyAssignment]"
+                },
+                "parameters": {
+                    "topLevelManagementGroupPrefix": {
+                        "value": "[parameters('enterpriseScaleCompanyPrefix')]"
+                    },
+                    "enforcementMode": {
+                        "value": "Default"
```

Comment:
Should this be conditional like:
![image](https://user-images.githubusercontent.com/41163455/217502230-9da6a955-4a42-458c-8800-2b2f846df0e5.png)